tests/testthat/test_featsel_praznik.R

```diff
@@ -15,14 +15,15 @@ test_that("filterFeatures_praznik", {
   for (candidate in candidates) {
     fv = generateFilterValuesData(task, method = candidate, nselect = 2L)
     expect_class(fv, "FilterValues")
-    # FIXME: CMIM and MIM return NA for factors for praznik >= v7.0.0
-    # upstream issue: https://gitlab.com/mbq/praznik/issues/19
-    if (candidate == "praznik_CMIM" || candidate == "praznik_MIM") {
-      expect_equal(sum(!is.na(fv$data$value)), 1)
+    # New CMIM engine since praznik v7.0.0
+    # a return value for 'c' was actually a bug for <= 7.0.0
+    # see https://gitlab.com/mbq/praznik/issues/19
+    if (candidate == "praznik_CMIM") {
+      expect_equal(sum(!is.na(fv$data$value)), 1, info = candidate)
     } else {
-      expect_equal(sum(!is.na(fv$data$value)), 2)
+      expect_equal(sum(!is.na(fv$data$value)), 2, info = candidate)
     }
-    expect_data_frame(fv$data, nrow = getTaskNFeats(task))
```

Comment:
Wait, how did this ever pass before with an incorrect argument name?

Reply:
partial matching

Reply:
Ugh.